dappnode_package.json

```
@@ -17,6 +17,7 @@
   },
   "links": {
     "host": "zcash.public.dappnode",
+    "apiEngine": "http://zcash.public.dappnode:8332",
```

```suggestion
    "api": "http://zcash.public.dappnode:8332",
```
just the RPC endpoint not an engineAPI